source/_docs/automation/templating.markdown

```
@@ -25,6 +25,7 @@ Triggers from all platforms will include the following data.
 
 | Template variable | Data |
 | ---- | ---- |
+| `trigger.alias` | Alias of the trigger.
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)